swagger-spec/registrations/children_list.yaml

```diff
@@ -0,0 +1,203 @@
+# /registrations/{node_id}/children/
+get:
+  summary: List all child nodes
+  description: >-
+
+    A paginated list of children of a registration.
+
+
+    The list consists of the next level child nodes for the given node.
+    The returned nodes are sorted by their `date_modified`, with the most recently updated child nodes appearing first.
+
+
+    The list will include child nodes that are public, as well as child nodes that are private, if the authenticated user has permission to view them.
+
+    #### Returns
+
+    Returns a JSON object containing `data` and `links` keys.
+
+
+    The `data` key contains an array of up to 10 child nodes.
+    If the given node has zero child nodes, the `data` key will contain an empty array.
+    Each resource in the array is a separate node object and contains the full representation of the child node, meaning additional requests to the child node's detail view are not necessary.
+
+
+    The `links` key contains a dictionary of links that can be used for [pagination](#Introduction_pagination).
+
+
+    This request should never return an error.
+
+    #### Filtering
+
+    You can optionally request that the response only include nodes that match your filters by utilizing the `filter` query parameter, e.g.
+    https://api.osf.io/v2/registrations/wucr8/children/?filter[title]=reproducibility.
+
+
+    Nodes may be filtered by their `id`, `title`, `category`, `description`, `public`, `tags`, `date_created`, `date_modified`, `root`, `parent`, `preprint`, and `contributors`.
```

Comment:
Let's remove `preprint` from this list. While technically true, it shouldn't be documented and attempting to do so causes a 502 on production right now.